assets/xml/objects/object_hanareyama_obj.xml

```diff
@@ -59,7 +58,10 @@
         <DList Name="object_hanareyama_obj_DL_003578" Offset="0x3578" />
         <Texture Name="object_hanareyama_obj_Tex_003638" OutName="tex_003638" Format="i4" Width="64" Height="64" Offset="0x3638" />
         <Texture Name="object_hanareyama_obj_Tex_003E38" OutName="tex_003E38" Format="rgba16" Width="32" Height="32" Offset="0x3E38" />
-        <!-- <Blob Name="object_hanareyama_obj_Blob_004638" Size="0x228" Offset="0x4638" /> -->
+        <Blob Name="object_hanareyama_obj_Blob_004638" Size="0xD8" Offset="0x4638" /> <!-- array of 54 displaylist pointers --> 
```

Comment:
Just want to note this. Does/Will ZAPD support this?

Reply:
It doesn't, dunno if it will.

Reply:
I made a PR to allow the proper extraction of those:
<https://github.com/zeldaret/ZAPD/pull/232>

Reply:
Should be addressed with ZAPD update.